css/style.css

```diff
@@ -16,4 +8,135 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+.logo-icon {
+  width: 30px;
+}
+
+.menu-nav {
+  list-style-type: none;
+  padding: 17px;
+  margin: 0px;
+  text-align: center;
+  font-size: 16px;
+}
+
+.nav-flex-box {
+  display: flex;
+  justify-content: center;
+  max-width: 100%;
+}
+
+.nav-list {
```

Comment:
Choose your naming carefully - you are using this class on each list _item_, not on the list itself. So it would be more appropriate to name this class `nav-list-item`.